sgx/src/enclave/enclave.rs

```diff
@@ -10,16 +10,14 @@ use mmap::Unmap;
 use crate::types::ssa::Exception;
 use crate::types::tcs::Tcs;
 
-/// Opcodes for non-privileged SGX leaf functions
-///
-/// TODO add more comprehensive docs
+/// How to enter an enclave
 #[repr(u32)]
 #[derive(Copy, Clone, Debug, PartialEq)]
-pub enum Leaf {
-    /// EAX = 2, `EENTER`
+pub enum Entry {
+    /// Enter an enclave normally
     Enter = 2,
 
-    /// EAX = 3, `ERESUME`
+    /// Resume an enclave after an asynchronous exit
```

Comment:
```suggestion
    /// Resume an enclave after an asynchronous exit.
    /// In a Keep, this is used after a proxied syscall is complete.
```

Reply:
This library doesn't know anything about keeps. So we can't mention them in documentation.

Reply:
@npmccallum Fair!